internal/server/statvar/statvar_group.go

```diff
@@ -114,6 +116,52 @@ func filterSVG(in *pb.StatVarGroups, statVars []string) *pb.StatVarGroups {
 	return result
 }
 
+// fixCustomRootDescendentStatVarCount returns the DescendentStatVarCount for node curID.
+// Mutates DescendentStatVarCount for all nodes in id2Node starting at customSvgRoot.
+func fixCustomRootDescendentStatVarCount(id2Node map[string]*pb.StatVarGroupNode, curID string) int32 {
```

Comment:
This in theory works even for non-custom hierarchy. So maybe name this to 

`adjustDescendentStatVarCount(nodeMap map[string]*pb. StatVarGroupNode, node string) int32`